instance/tasks.py

```diff
@@ -61,33 +61,19 @@ def spawn_appserver(
     Optionally tag the instance with 'success_tag' when the deployment succeeds,
     or failure_tag if it fails.
     """
-    for i in range(1, num_attempts + 1):
-        logger.info('Retrieving instance: ID=%s', instance_ref_id)
-        # Fetch the instance inside the loop, in case it has been updated
```

Comment:
This is a kind of behaviour change. Probably not a big deal though.

Reply:
This behaviour was moved to the instance model [here](https://github.com/open-craft/opencraft/blob/1cfc0f53656555708fb397c4b067ba46c645988a/instance/models/openedx_instance.py#L247).

Reply:
What I meant was that earlier the instance was being fetched for each run of the loop, and that is no longer the case. So if the instance changes between two loops, the old behaviour would be to use the latest configuration from the instance, while the new behaviour will use the old config that was loaded initially. 

Probably not a big deal though.